libp2p/NodeTable.cpp

```diff
@@ -482,6 +490,20 @@ void NodeTable::onPacketReceived(
 
             case FindNode::type:
             {
+                auto const sourceNodeEntry = nodeEntry(packet->sourceid);
+                if (!sourceNodeEntry)
+                {
+                    LOG(m_logger) << "Source node (" << packet->sourceid << "@" << _from
+                                  << ") not found in node table. Ignoring FindNode request.";
+                    return;
+                }
+                if (sourceNodeEntry->lastPongReceivedTime + c_bondingTimeSeconds <
+                    RLPXDatagramFace::secondsSinceEpoch())
+                {
+                    LOG(m_logger) << "Endpoint proof has expired! Ignoring FindNode request";
```

Comment:
Not quite precise, could be also that we haven't received any proof yet.

Reply:
Good point...maybe a better log message would be something like “unexpected FindNode request”

Reply:
Going to split this into 2 cases and log 2 different messages - 1 where we haven't performed the endpoint proof yet (lastPongReceivedTime == 0) and 1 where the endpoint proof has expired.